### DISCUSSION TITLE
Tailwind Newbie

### DISCUSSION THREAD

**Discussion by rkingg-dev** [tailwindlabs/tailwindcss#11978] [Help]
Hello Tailwind enthusiasts,

I'm planning to work on a personal project utilizing Tailwind CSS. I've got a personal license and noticed that there are both dark and light mode components available.

Now, here's my question: When I create the HTML layout for my project, how can I make it adapt to the dark mode?

I'm a bit of a newbie when it comes to web apps, so any guidance would be greatly appreciated. Thanks!

**Comment by eYinka** [comment]
Tailwind CSS supports dark mode out of the box as per [docs](https://tailwindcss.com/docs/dark-mode)! 

However, you would need to probably implement a switch and/or use the browser's current theme. Building the switch that helps the user toggle between light and dark mode requires some javascript though. How you go about it precisely depends on what framework or library you are using in your project. Here's a nicely written tutorial that shows how you can do such using plain HTML and JavaScript: https://www.freecodecamp.org/news/how-to-build-a-dark-mode-switcher-with-tailwind-css-and-flowbite/

I hope this helps!
